app/src/main/res/values/strings.xml

```diff
@@ -1,4 +1,16 @@
 <resources>
     <string name="app_name">HungryWolfes</string>
-    <string name="title">Delicious food for you</string>
+
+    <string name="home_screen_title">Delicious food for you</string>
+    <string name="search">Search</string>
+
+    <string name="MenuHome">Home</string>
```

Comment:
Resource names should be lowercase and with underscores as a general rule 

`menu_home`

Reply:
I've changed that already and also did a commit with it